src/gui/shell.cpp

```diff
@@ -858,21 +864,27 @@ void Shell::handleNeovimNotification(const QByteArray &name, const QVariantList&
 
 			qDebug() << "Neovim changed clipboard" << data << type << reg_name << clipboard;
 			QGuiApplication::clipboard()->setMimeData(clipData, clipboard);
-		} else if (guiEvName == "WindowOpacity" && args.size() == 2) {
+		}
+		else if (guiEvName == "WindowOpacity" && args.size() == 2) {
 			bool ok = false;
 			auto val = args.at(1).toDouble(&ok);
 			if (ok) {
 				emit neovimOpacity(val);
 			}
-		} else if (guiEvName == "ShowContextMenu") {
+		}
+		else if (guiEvName == "ShowContextMenu") {
 			emit neovimShowContextMenu();
-		} else if (guiEvName == "AdaptiveColor") {
+		}
+		else if (guiEvName == "AdaptiveColor") {
 			handleGuiAdaptiveColor(args);
-		} else if (guiEvName == "AdaptiveFont") {
+		}
+		else if (guiEvName == "AdaptiveFont") {
 			handleGuiAdaptiveFont(args);
-		} else if (guiEvName == "AdaptiveStyle") {
+		}
+		else if (guiEvName == "AdaptiveStyle") {
 			handleGuiAdaptiveStyle(args);
-		} else if (guiEvName == "AdaptiveStyleList") {
+		}
+		else if (guiEvName == "AdaptiveStyleList") {
```

Comment:
nitpick:

Please keep the diff to a minimum. These lines are changing, so they can be reverted.

Reply:
All these come from the patch that the CI provided for clang-format. I did not change these files. The build was rejected because of the formatting check.

Reply:
Ahhhh, clang-format is being weird... That is a new check.

Let me investigate.

Reply:
Ohhh, and feel free to ignore the `clang-format` and CodeCov tests. Consider them both soft failures.

Clang-format is new, and I am still ironing out the bugs. Open to any suggestions.

Personally, I don't like:
 - All of the header changes/re-ordering.
 - The modifications to un-changed lines here.

Reply:
Dropped the commit with the formatting changes.